sentry-android-core/src/main/java/io/sentry/android/core/AndroidOptionsInitializer.java

```diff
@@ -27,6 +27,7 @@
  * Android Options initializer, it reads configurations from AndroidManifest and sets to the
  * SentryOptions. It also adds default values for some fields.
  */
+@SuppressWarnings("Convert2MethodRef")
```

Comment:
I always wonder what/why are we suppressing things here. There's no spot to add a 'reason', right? 
Would it make sense to add as a comment?

```suggestion
@SuppressWarnings("Convert2MethodRef") // Because of reasons
```